queries.sql

```diff
@@ -7,4 +7,69 @@ SELECT date_of_birth FROM animals  WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 't';
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN TRANSACTION;
+SAVEPOINT SP1;
+UPDATE animals
+SET species = 'undefined';
+ROLLBACK TO SP1;
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235706361-67bedf3b-b595-4086-a45c-25aeff38d096.png)

- [ ] It would be very nice if you follow all the requirements and fix the following things 👍 
1. First you have set up a savepoint and you are rollbacking to the save point so it means that your transaction is not roll backed completely 🤔
2. Second, kindly verify the changes using Select statements as required by the project requirements 😊

After making changes your transaction will look like below ⬇️

```SQL
   BEGIN TRANSACTION;
   UPDATE animals
   SET species = 'undefined';
   SELECT species from animals; -- verify that change was made
   ROLLBACK;
   SELECT species from animals; -- verify that change was undone

```